### ISSUE TITLE
Get every SonarJS rules enabled

### ISSUE THREAD

**Issue by tassin-gauthier** [SonarSource/eslint-plugin-sonarjs#430]
**I want to report a bug.**

When I use SonarJS in a vue.js project, some issues are detected by eslint with sonarJS rules, for example [sonarjs/prefer-immediate-return](https://github.com/SonarSource/eslint-plugin-sonarjs/blob/master/docs/rules/prefer-immediate-return.md) is triggered. But some rules aren't triggering an error by SonarJS, like the rule css:S4666, but SonarCloud detect it as an issue.

So, I know SonarJS is working because it can detect some error, but is there something to do to enable every rules ? I think https://stylelint.io/user-guide/rules/no-duplicate-selectors/ is the example of the rule who "doesn't exist", but i read that SonarJS was including stylelint... ?

My eslint config : 
```js
module.exports = {
  root: true,
  env: {
    node: true,
    es2022: true,
  },
  extends: [
    "plugin:vue/vue3-essential",
    "plugin:vue/vue3-strongly-recommended",
    "plugin:vue/vue3-recommended",
    "plugin:vue-scoped-css/vue3-recommended",
    "eslint:recommended",
    "@vue/typescript/recommended",
    "plugin:prettier/recommended",
    "plugin:sonarjs/recommended",
  ],
  parserOptions: {
    ecmaVersion: 2020,
  },
  rules: {
    "no-console": ["warn", { allow: ["error"] }],
    "no-debugger": "warn",
  },
};
```

**eslint-plugin-sonarjs version:** 0.23.0<br>
**eslint version:** 8.45.0<br>
**Node.js version:** 20.8.0<br>

**Rule key:** [css:S4666](https://sonarcloud.io/organizations/dktunited/rules?open=css%3AS4666&rule_key=css%3AS4666)

**Comment by vdiez** [comment]
Hello @tassin-gauthier,

can you clarify what you mean by using SonarJS? This repo is just a plugin for eslint and does not contain all SonarJS rules (repo [here](https://github.com/SonarSource/SonarJS)), just a selection. In order to use all rules (like the CSS rules) in your IDE one must use SonarLint.

Does this answer your question?

Cheers

**Comment by tassin-gauthier** [comment]
Well ok I'm gonna use sonarlint too, thanks for your answer
